<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeliculasWindow</class>
 <widget class="QMainWindow" name="PeliculasWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="Peliculas">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <property name="rowCount">
     <number>5</number>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Título</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Actor</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Sinopsis</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Recaudación</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Extras</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Ficha</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="PeliculasComunes">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>411</width>
      <height>101</height>
     </rect>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>true</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>350</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Título</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Año</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Películas Comunes</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelPoster">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>141</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>Posters</string>
    </property>
    <property name="textFormat">
     <enum>Qt::MarkdownText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelTituloActores">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>411</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;Resultado de la búsqueda Actores&quot;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTituloTitulo2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>411</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;Resultados de la búsqueda Titulo&quot;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPeliculas">
    <property name="title">
     <string>Resultado de Busqueda</string>
    </property>
   </widget>
   <addaction name="menuPeliculas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
